### DISCUSSION TITLE
Question about background-image

### DISCUSSION THREAD

**Discussion by ronaldohoch** [GrapesJS/grapesjs#3808] [Q&A] [answered]
Hello!

I have a custom `open-assets` command, that opens a custom modal to a own assets manager and this is the part of the code that add the image url to the element...

```javascript
  if(stateEditor==="EMAIL"){
      // MJML usa background-url para setar o background de um elemento
      editor.getSelected().addStyle({ 'background-url': textUrl });
  }else{
      // Setar background em um elemento com html normal
      editor.getSelected().addStyle({ 'background-image': `url(${textUrl})` });
  }
```

But after that, the trait didn't update.
![image](https://user-images.githubusercontent.com/2287371/134409373-039bb36c-1c3a-4749-9d40-461c85abe273.png)

Is there a built-in command to update it?

See, if i click in other and get back to it, the background is there.
![image](https://user-images.githubusercontent.com/2287371/134409485-1fd29a50-5fe8-43b6-9c46-9fcedfd35346.png)

**Comment by artf** [comment, marked as answer]
Try to trigger this post style update `editor.trigger('component:toggled')`

**Reply by ronaldohoch** [reply]
Worked!
